### DISCUSSION TITLE
What facial orientation picture does Build Checkpoints tool need？

### DISCUSSION THREAD

**Discussion by sdcarterchen** [glucauze/sd-webui-faceswaplab#54] [Q&A]
Does **Build Checkpoints tool** only require face pictures facing the front? Or does it require facial orientation pictures from various angles? Is there a resolution requirement?

**Comment by glucauze** [comment]
There is no absolute truth to this. You can use any image at any resolution. A combination of these and even a combination of different faces. Sometimes you get very good results with several orientations, sometimes not. :)

The best way to test this is to check the override option.
